torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -457,13 +616,16 @@ def identify_mutated_tensors(kernel, kwargs):
         import traceback
 
         warnings.warn(
-            "Encountered an exception in identify_mutated_tensors, assuming every input is mutated"
-        )
-        log.debug(
+            "Encountered an exception in identify_mutated_tensors, "
+            "assuming every input is mutated:\n"
             "".join(
                 traceback.TracebackException.from_exception(e).format()  # noqa: G001
             )
         )
+        if ttir_module is not None:
+            log.debug("TTIR: %s", str(ttir_module))
+        if functions is not None:
+            log.debug("functions: %s", str(functions))
```

Comment:
str of functions would be hard to read, so i'd probably do something like
```suggestion
            for name, function in functions.items():
                log.debug(f"===\t{name}\t===")
                for ret, ops in function.items():
                    log.debug(f"{ret}\t=>\t{ops}")
```

Reply:
Sure.